dlk/python/dlk/core/data_types.py

```diff
@@ -150,6 +222,16 @@ def nptype(cls):
         return np.uint64
 
 
+class PackedUint64(Primitive, int):
+    @classmethod
+    def cpptype(cls):
+        return 'QuantizedPacked<uint64_t>'
+
+    @classmethod
+    def nptype(cls):
+        return np.uint64
+
+
```

Comment:
would prefer to not have types which we do not use

it is easy to add missed ones when we start using them and keeping unused code will increase maintenance effort / mental load while reading.

Reply:
Remove these types except `PackedUint64`.